README.md

```diff
@@ -329,12 +329,15 @@ This tool wraps underlying license / copyright scanners with a common API so all
 same way to easily run them and compare their results. If passed an ORT result file with an analyzer result (`-i`), the
 _scanner_ will automatically download the sources of the dependencies via the _downloader_ and scan them afterwards.
 
-Currently, the following license scanners are supported:
+We recommend to use ORT with one of the following scanners as their integration is tested best:
+
+* [ScanCode](https://github.com/nexB/scancode-toolkit)
+
+Additionally, the following reference implementations exist to showcase more scanner integrations:
```

Comment:
Replace "to showcase more scanner integrations" with "as a showcase to demonstrate ORT can support other scanners:"

Reply:
I don't like the "other scanners" wording here so much, as it sounds a bit like "we do not only support ScanCode as a single scanner, but also others". While that's actually correct saying so, it would sound a bit odd IMO as soon as we add more scanners to the "has been thoroughly tested" section. As such I'd prefer to stick with my wording.

Reply:
I not a fan of the word "more" in your sentence. What if we simply shorten the sentence to "Additionally, the following reference implementations exist:" 

For me "reference" and "showcase" are similar in their meaning in this sentence.

Reply:
Fine with me, done.